battery.sh

```diff
@@ -1,4 +1,4 @@
 pmset -g batt |
 grep -Eo "\d+%" |
 cut -d% -f1 |
-tr -d "\n"
+tr -d "\n" ; top -u -l 1 | head -4
```

Comment:
does this actually work? I have logged event and it only shows `100` (battery percentage) as the event output.

Reply:
I'm seeing this work, copying it into terminal works too. I tried to get this to be on its own line but it needed to be on this line.

Reply:
I've seen issues when switching branches. Can you restar Ubersicht and see if that helps?

Reply:
Ah yes, this was my mistake. Thank you.